schemas/snuba-items.v1.schema.json

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "item",
+  "type": "object",
+  "properties": {
+    "organization_id": {
+      "$ref": "#/definitions/UInt"
+    },
+    "project_id": {
+      "$ref": "#/definitions/UInt"
+    },
+    "trace_id": {
+      "$ref": "#/definitions/UUID"
+    },
+    "item_id": {
+      "$ref": "#/definitions/UInt"
+    },
+    "item_type": {
+      "type": "integer",
+      "enum": [1, 2, 3]
+    },
+    "attributes": {
+      "$ref": "#/definitions/Attributes"
+    },
```

Comment:
Do we want to use OTel-ish attribute objects (similar to [ourlogs](https://github.com/getsentry/sentry-kafka-schemas/blob/746ca5aa685f0d64cf7e553f8221196d8cbebd76/schemas/snuba-ourlogs.v1.schema.json#L49-L72) ) or a simple KV? 

I suppose if we do protobuf we don't need to worry about typing w/ json as much